index.js

```diff
@@ -67,7 +67,7 @@ async function getTransaction(hash, network) {
             let networkDetails = config.networks.testnet;
             if (network === 'main') networkDetails = config.networks.main;
 
-            const provider = new ProxyProvider(networkDetails.provider, timeOut);
+            const provider = new ProxyProvider(networkDetails.provider, {timeOut});
```

Comment:
@Minhaj-ul-abidin the object here should be `{timeout:timeOut}` . Can you please check this once.